src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -840,6 +834,12 @@ void UpdatingGestureRecognizers()
 				return;
 			}
 
+			if (!hasPointerGesture)
+			{
+				_subscriptionFlags |= SubscriptionFlags.ContainerPgrPointerEventsSubscribed;
+				SubscribePointerEvents(_container);
+			}
```

Comment:
I had to read the code a couple times to understand why `SubscribePointerEvents()` was called twice. Could there be a comment just saying briefly why, something like: "Pan, pinch, and swipe gestures need pointer events if not subscribed yet".